### PR TITLE
0.8.2

### PR DESCRIPTION
**Pull request by gokhanbarisaker**

### CODE REVIEW

**Comment by paddybyers** [comment]
LGTM

File: pom.xml [hunk 1]
```diff
@@ -15,7 +15,7 @@
   <licenses>
     <license>
       <name>The Apache Software License, Version 2.0</name>
-      <url>https://raw.github.com/ably/ably-java/master/LICENSE</url>
+      <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
```
Comment (mattheworiordan) [inline pom.xml]: I don't mind this change, but why?

**Comment by mattheworiordan** [comment]
One small comment, but otherwise LGTM

File: pom.xml (hunk 1 above)
Comment (gokhanbarisaker) [inline pom.xml]: copy/paste error, a small glitch in the matrix

**Comment by gokhanbarisaker** [comment]
done

**Pull request merged**